src/Makefile.am

```diff
@@ -83,33 +82,35 @@ extraLDADD    += -laws4c
 extraLDADD    += -laws4c_extra
 endif
 
-ior_SOURCES += $(extraSOURCES)
-ior_LDFLAGS += $(extraLDFLAGS)
-ior_LDADD += $(extraLDADD)
+ior_SOURCES  += $(extraSOURCES)
+ior_LDFLAGS  += $(extraLDFLAGS)
+ior_LDADD    += $(extraLDADD)
 ior_CPPFLAGS += $(extraCPPFLAGS)
 
-mdtest_SOURCES += $(extraSOURCES)
-mdtest_LDFLAGS += $(extraLDFLAGS)
-mdtest_LDADD += $(extraLDADD)
+mdtest_SOURCES  += $(extraSOURCES)
+mdtest_LDFLAGS  += $(extraLDFLAGS)
+mdtest_LDADD    += $(extraLDADD)
 mdtest_CPPFLAGS += $(extraCPPFLAGS)
 
 IOR_SOURCES  = $(ior_SOURCES)
+IOR_LDFLAGS  = $(ior_LDFLAGS)
 IOR_LDADD    = $(ior_LDADD)
-IOT_CPPFLAGS = $(ior_CPPFLAGS)
+IOR_CPPFLAGS = $(ior_CPPFLAGS)
 
 MDTEST_SOURCES  = $(mdtest_SOURCES)
+MDTEST_LDFLAGS  = $(mdtest_LDFLAGS)
 MDTEST_LDADD    = $(mdtest_LDADD)
 MDTEST_CPPFLAGS = $(mdtest_CPPFLAGS)
 
 libaiori_a_SOURCES += $(extraSOURCES)
-#libaiori_a_LIBADD += $(extraLDFLAGS) $(extraLDADD)
-
+libaiori_a_LIBADD   =
```

Comment:
Can we remove this variable if it's not set?

Reply:
I'll do that yes.